.github/workflows/build-ampd-handlers-release.yaml

```diff
@@ -0,0 +1,245 @@
+name: ampd-handlers -  Build and release binary and image
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: Github handler tag to release binaries for (e.g., evm-handler-v1.0.0, sui-handler-v1.0.0)
+        required: true
+        default: latest
+      dry-run:
+        description: Run in dry-run mode (set to false to actually release)
+        required: true
+        default: 'true'
+
+jobs:
+  extract-semver:
+    runs-on: ubuntu-22.04
+    name: Validate tag and extract semver
+    permissions:
+      contents: write
+      packages: write
+      id-token: write
+    outputs:
+      semver: ${{ steps.extract_semver.outputs.semver }}
+      version: ${{ steps.extract_semver.outputs.version }}
+      handler: ${{ steps.extract_semver.outputs.handler }}
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@7474bc4690e29a8392af63c5b98e7449536d5c3a # v4
+        with:
+          aws-region: us-east-2
+          role-to-assume: arn:aws:iam::${{ secrets.AWS_ACCOUNT_ID }}:role/ghwf-${{ github.event.repository.name }}
+
+      - name: Extract semver and handler from tag
+        id: extract_semver
+        run: |
+          tag="${{ github.event.inputs.tag }}"
+          handler=$(echo "$tag" | sed -E 's/-handler-v.*//')
+          full_semver=$(echo "$tag" | sed -E 's/.*-handler-(v.*)/\1/')
+          version_number=$(echo "$full_semver" | sed 's/^v//')
+          
+          if [ -z "$handler" ] || [ -z "$full_semver" ]; then
+            echo "Error: Failed to extract handler or semver from tag: $tag"
+            exit 1
+          fi
+          
+          echo "handler=$handler" >> $GITHUB_OUTPUT
+          echo "semver=$full_semver" >> $GITHUB_OUTPUT
+          echo "version=$version_number" >> $GITHUB_OUTPUT
+
+      - name: Validate tag
+        env:
+          TAG: ${{ github.event.inputs.tag }}
+          HANDLER: ${{ steps.extract_semver.outputs.handler }}
+          SEMVER: ${{ steps.extract_semver.outputs.semver }}
```

Comment:
are they already set at this point? I'm not super familiar with how job outputs work